.travis.yml

```diff
@@ -10,6 +10,7 @@ python:
   - "3.6"
 install:
   - npm install jsdoc@3.5.5
+  - npm install typedoc
```

Comment:
We should probably pin it so tests don't cry wolf someday because typedoc changed behavior.